Formula/i/instaloader.rb

```diff
@@ -0,0 +1,44 @@
+class Instaloader < Formula
+  include Language::Python::Virtualenv
+
+  desc "Download media from Instagram"
+  homepage "https://instaloader.github.io/"
+  url "https://github.com/instaloader/instaloader/archive/refs/tags/v4.10.1.tar.gz"
+  sha256 "5a2e0519e44bd4005363adc7d4f88ed5967abd9457e119dee180a33c77cc4890"
+  license "MIT"
+
+  depends_on "python@3.11"
+
+  resource "certifi" do
+    url "https://files.pythonhosted.org/packages/98/98/c2ff18671db109c9f10ed27f5ef610ae05b73bd876664139cf95bd1429aa/certifi-2023.7.22.tar.gz"
+    sha256 "539cc1d13202e33ca466e88b2807e29f4c13049d6d87031a3c110744495cb082"
+  end
```

Comment:
we can vendor this out using `python-certifi` (also need to update pypi mapping file)